invenio_app_ils/ext.py

```diff
@@ -209,7 +209,7 @@ def init_app(self, app):
         )
         app.register_blueprint(
             Blueprint(
-                "invenio_app_ils_circulation_ mail",
+                "invenio_app_ils_circulation_mail",
```

Comment:
Wait this is outdated no? I remember adding this through an entrypoint in the end, no?

Reply:
The entrypoint didn't work.